src/main/java/org/c4sg/mapper/OrganizationMapper.java

```diff
@@ -17,4 +22,11 @@ public Organization getOrganizationEntityFromDto(OrganizationDTO organizationDTO
 		Organization organization = map(organizationDTO, Organization.class);
 		return organization;
 	}
+	
+	public OrganizationDTO getOrganizationDtoFromEntity(UserOrganization userOrganization){
```

Comment:
That's right. Happens when you just copy from an existing one that does exactly the same function. We shall reuse.